plutus-playground-client/test/BridgeTests.purs

```diff
@@ -27,6 +48,18 @@ all =
 jsonHandling :: forall eff. TestSuite (exception :: EXCEPTION, fs :: FS, random :: RANDOM | eff)
 jsonHandling = do
     suite "Json handling" do
+      test "Decode a List." do
+        assertDecodesTo
+          (Proxy :: Proxy (List TokenId))
+          "test/token_ids.json"
+      test ("Decode a NonEmptyList.") do
+        assertDecodesTo
+          (Proxy :: Proxy (NonEmptyList TokenId))
+          "test/token_ids.json"
+      test "Decode a KnownCurrency." do
+        assertDecodesTo
+          (Proxy :: Proxy KnownCurrency)
+          "test/known_currency.json"
```

Comment:
worth testing a non-nonempty list?

Reply:
I mean try to decode an empty list to a non-empty list, maybe it's not worth the test TBH because the type makes it impossible.

Reply:
Yeah, I like that. I'll add it. :-)